app/controllers/api_controller.rb

```diff
@@ -11,6 +13,17 @@ def not_found
   end
 
   def set_locale
-    I18n.locale = current_api_v1_user.locale
+    I18n.locale = current_user.locale
+  end
+
+  def pundit_user
+    current_user
+  end
+
+  private
+
+  def user_not_authorized
+    flash[:warning] = 'You are not authorized to perform this action.'
```

Comment:
```suggestion
    policy_name = exception.policy.class.to_s.underscore
    message = t "#{policy_name}.#{exception.query}", scope: 'pundit', default: :default
    render json: { error: message }, status: :forbidden
```
and add locales for pundit